examples/pylab_examples/plotfile_demo.py

```diff
@@ -1,35 +1,47 @@
-from pylab import plotfile, show, gca
+import matplotlib.pyplot as plt
+import numpy as np
+
 import matplotlib.cbook as cbook
 
 fname = cbook.get_sample_data('msft.csv', asfileobj=False)
 fname2 = cbook.get_sample_data('data_x_x2_x3.csv', asfileobj=False)
 
 # test 1; use ints
-plotfile(fname, (0, 5, 6))
+plt.plotfile(fname, (0, 5, 6))
 
 # test 2; use names
-plotfile(fname, ('date', 'volume', 'adj_close'))
+plt.plotfile(fname, ('date', 'volume', 'adj_close'))
 
 # test 3; use semilogy for volume
-plotfile(fname, ('date', 'volume', 'adj_close'), plotfuncs={'volume': 'semilogy'})
+plt.plotfile(
+    fname,
+    ('date',
+     'volume',
+     'adj_close'),
+    plotfuncs={
+        'volume': 'semilogy'})
```

Comment:
7 lines for the ``plot_file`` function is a bit too much. Please put it in 2 lines. 

```
plt.plotfile(fname, ('date', 'volume', 'adj_close'), 
             plotfuncs={'volume': 'semilogy'})
```